modules/data-entry/src/main/frontend/src/questionnaire/FormPagination.jsx

```diff
@@ -51,7 +51,22 @@ class Page {
  * Component that displays a page of a Form.
  */
 function FormPagination (props) {
-  let { classes, enabled, variant, navMode, saveInProgress, lastSaveStatus, setPagesCallback, isPageCompleted, enableSave, onDone, doneLabel, doneIcon, questionnaireData, disableProgress, onPageChange } = props;
+  let {
+    classes,
+    enabled = true,
+    variant = "progress",
+    navMode = "back_next",
+    lastSaveStatus = true,
```

Comment:
`saveInProgress` got removed.
```suggestion
    saveInProgress,
    lastSaveStatus = true,
```